server/server.go

```diff
@@ -605,6 +605,46 @@ func (s *Server) v1DiagnosticsGet(w http.ResponseWriter, r *http.Request) {
 	writer.JSON(w, 200, resp, pretty)
 }
 
+func (s *Server) v1VersionGet(w http.ResponseWriter, r *http.Request) {
+
+	code := 200
+
+	opaVersion := `
+{
+  "result": {
+    "Version": "{{.Version}}",
+    "BuildCommit": "{{.BuildCommit}}",
+    "BuildTimestamp": "{{.BuildTimestamp}}",
+    "BuildHostname": "{{.BuildHostname}}"
+  }
+}
+	`
+	tmpl, err := template.New("").Parse(opaVersion)
+	if err != nil {
+		panic(err)
+	}
+
+	var buf bytes.Buffer
+
+	err = tmpl.Execute(&buf, struct {
+		Version        string
+		BuildCommit    string
+		BuildTimestamp string
+		BuildHostname  string
+	}{
+		Version:        version.Version,
+		BuildCommit:    version.Vcs,
+		BuildTimestamp: version.Timestamp,
+		BuildHostname:  version.Hostname,
+	})
+
+	jsonBytes := buf.Bytes()
```

Comment:
🤔 A template, interesting. I'd have expected something like this (sorry, untested)
```go
m := struct {
	Version, BuildCommit, BuildTimestamp, BuildHostname string
}{
	Version:        version.Version,
	BuildCommit:    version.Vcs,
	BuildTimestamp: version.Timestamp,
	BuildHostname:  version.Hostname,
}
err := json.NewEncoder(w).Encode(&m) // alternatively, using a buffer and writer.Bytes like done here
// TODO: handle err
```
...but this should do the trick, too, of course.

Reply:
In repl.go there is already a template and I wanted to keep both outputs exactly the same. I think in a future commit the idea would be to move the template to version package since that's where all the data is inserted during build. 

Then both repl.go and server.go could use it from versio.go,